backend/de.metas.banking/de.metas.banking.camt53/src/main/java/de/metas/banking/camt53/BankStatementCamt53Service.java

```diff
@@ -332,6 +346,72 @@ private Optional<BankStatementLineCreateRequest> buildBankStatementLineCreateReq
 		return Optional.of(bankStatementLineCreateRequestBuilder.build());
 	}
 
+
+	@NonNull
+	private List<BankStatementLineCreateRequest> buildBankStatementLineCreateRequestForMultiBatch(@NonNull final ImportBankStatementLineRequest importBankStatementLineRequest)
+	{
+		final List<BankStatementLineCreateRequest> lineRequests =  new ArrayList<>();
+
+		// create request for summary line
+		buildBankStatementLineCreateRequest(importBankStatementLineRequest)
+				.ifPresent(lineRequests::add);
+
+		final IStatementLineWrapper entryWrapper = importBankStatementLineRequest.getEntryWrapper();
+		final OrgId orgId = importBankStatementLineRequest.getOrgId();
+
+		final ZonedDateTime statementLineDate = entryWrapper.getStatementLineDate(orgDAO.getTimeZone(orgId))
```

Comment:
since getStatementLineDate is taking zoneId as parameter, why the hell is not returning ZonedDateTime ?!